hub/api/config/db-migration/15-db-cronjob.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hub-sync-db
+  namespace: tekton-hub-preview
+  labels:
+    app: api
+spec:
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: sync-db
+              image: byrnedo/alpine-curl:0.1.8
```

Comment:
please take note when using images. it is better to use official images whenever possible - https://hub.docker.com/r/curlimages/curl